qcelemental/tests/test_constants.py

```diff
@@ -5,69 +5,110 @@
 
 import qcelemental
 
+_pc_default = qcelemental.constants.name[-4:]
 
-def test_access_1a():
-    assert qcelemental.constants.c == 299792458
 
+@pytest.fixture(scope="module")
```

Comment:
Ah:
```python
@pytest.fixture(scope="module", params=[None, "CODATA2014", "CODATA2018"])
def contexts(request):
    if request.param:
        return qcelemental.PhysicalConstantsContext(request.param)
    else:
        return qcelemental.constants
```

You will no longer need items like: `@pytest.mark.parametrize("context", ["default", "2014", "2018"])`.

Reply:
nifty! if only one could get at the param other than through the id. anyways, test_constants.py patched up now.